Components/Model/Product.php

```diff
@@ -121,6 +124,22 @@ public function loadData(Article $article, Shop $shop = null)
         );
     }
 
+    /**
+     * Add Sku variations to the current article
+     * @param Article $article
+     * @return \Nosto\Object\Product\SkuCollection
+     */
+    public function amendSkus(\Shopware\Models\Article\Article $article, Shop $shop)
```

Comment:
We have been using `amendSomething` when the method directly populates attributes, in this case SKUs. I would call this `buildSkus` or change the method to directly populate SKUs

Reply:
Fixed! Thanks 😃